quadrat/sass/blocks/_table.scss

```diff
@@ -32,4 +32,4 @@
 			background-color: var(--wp--preset--color--tertiary);
```

Comment:
Looks like the stripy bit wasn't removed from Quadrat (or I think Zoologist and Geologist as there are still references to `.wp-block-table.is-style-stripes` in all of those three).